server/routes/health.js

```diff
@@ -1,36 +1,67 @@
+/* eslint-disable no-console */
 import express from 'express';
+import superagent from 'superagent';
+import config from '../../server/config';
 
 export default function createRouter(db, appInfo) {
   const router = express.Router();
 
+  function dbCheck() {
+    return db.select(db.raw('1'))
+      .then(() => ({ name: 'db', status: 'OK', message: 'OK' }))
+      .catch(err => ({ name: 'db', status: 'ERROR', message: err.message }));
+  }
+
+  function viperRestServiceCheck() {
+    return new Promise((resolve, reject) => {
+      if (process.env.USE_VIPER_SERVICE === 'false') {
+        resolve({ name: 'viperRestService', status: 'OK', message: 'Not enabled' });
+      }
+
+      superagent
+        .get(`${config.viperRestServiceHost}/analytics/health`)
+        .set('API-key', config.viperRestServiceAuthenticationKey)
+        .timeout({
+          response: process.env.VIPER_SERVICE_CONNECTION_TIMEOUT || 2000,
```

Comment:
Could these live in the config file?
`VIPER_SERVICE_CONNECTION_TIMEOUT ` and `VIPER_SERVICE_READ_TIMEOUT `